Extension/package.nls.json

```diff
@@ -19,5 +19,7 @@
     "c_cpp.command.takeSurvey.title": "Take Survey",
     "c_cpp.command.buildAndDebugActiveFile.title": "Build and Debug Active File",
     "c_cpp.command.logDiagnostics.title": "Log Diagnostics",
-    "c_cpp.command.rescanWorkspace.title": "Rescan Workspace"
+    "c_cpp.command.rescanWorkspace.title": "Rescan Workspace",
+    "c_cpp.command.VcpkgClipboardInstallSuggested.title": "Copy vcpkg install command to clipboard",
```

Comment:
This should be "title" case. And translated for zh-ch and it languages. I think Bob might know how to translate.